### PR TITLE
added --iteration command option for test cases that may need to run multiple times

### PR DESCRIPTION
**Pull request by jianzzha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,11 @@ cd sriov/tests/
 pytest -v SR_IOV_Permutation
 ```
 
+Certain test cases support multiple iterations of run. The number of interations can be specified using `--iteration` option. For example, to run the test case  `SR_IOV_MultipleVFCreation_withMTU` 10 times,
```
Comment (dkosteck) [inline README.md:86]: The first sentence can just be "Certain test cases support multiple iterations." Also in the second sentence it should be "iterations", just a small typo.

**Review by dkosteck — CHANGES_REQUESTED** [review]
The implementation looks good, not sure if the else case is redundant if the addoption function has a default storage of 1 (i.e. does this always populate the config with the option if nothing is passed? If so then the else case in pytest_generate_tests will never execute). Small typos in the documentation should be addressed before approval

File: README.md (hunk 1 above)
Comment (jianzzha) [reply README.md]: ack

**Review by akiselev1 — APPROVED** [review]
Looks good after Daniel's comments

**Review by dkosteck — APPROVED** [review]

**Pull request merged**